vendors/circom/circomlib/wtns/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("@kroma_network_tachyon//bazel:tachyon_cc.bzl", "tachyon_cc_library", "tachyon_cc_unittest")
+
+package(default_visibility = ["//visibility:public"])
+
+tachyon_cc_library(
+    name = "wtns",
+    srcs = ["wtns.cc"],
+    hdrs = ["wtns.h"],
```

Comment:
How about renaming `wtns` to `witness`?
If you agree this suggestion, please change every related name.

Reply:
Actually `zkey`, `r1cs` and `wtns` are the file extensions used by circom. So I think it's better to leave it as `wtns`!